config/runtime.exs

```diff
@@ -29,4 +29,5 @@ config :screenplay,
   slack_webhook_url: System.get_env("SLACK_WEBHOOK_URL"),
   environment_name: System.get_env("ENVIRONMENT_NAME"),
   sentry_frontend_dsn: System.get_env("SENTRY_DSN"),
-  api_v3_key: System.get_env("API_V3_KEY")
+  api_v3_key: System.get_env("API_V3_KEY"),
+  alerts_ui_url: System.get_env("ALERTS_UI_URL")
```

Comment:
Just a thought, but could the URLs live in this file (or some other config file) where they can just be conditionally chosen based on `environment_name`? I know there was a convo in Slack about the best place to put things like this, so this may push back on what was decided there. But `environment_name` is ultimately what we use to get the API URL, so we could do something similar with this. This would allow us to set `environment_name` and all other URLs will be picked correctly according to the value we use there.